docs/running.md

```diff
@@ -1,6 +1,15 @@
 Running bees
 ============
 
+Simplest Case
+-------------
+
+If you just want to run bees manually on a single btrfs partition that has no sub-volumes, the following bare bones solution should just work. 
+<path-to-bees>/bees  <path-to-btrfs-partition>
```

Comment:
This doesn't create the `.beeshome` directory or the `beeshash.dat` hash file.

I agree it _should_ work that way, but some supporting code changes will be required.  ;)

Reply:
@Zygo Since you ripped almost all the multi-context code out of bees now, it should be able to add a simple runtime option that would create the directory and the hash table with some sane default size.